<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>221</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>MENU</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="menu1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);
border-radius: 10px;
border: none;
</string>
      </property>
      <property name="text">
       <string>Giám sát trực tiếp</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="menu2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lịch sử vào ra</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="menu3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Danh sách người sử dụng</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="menu4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Nhiệt độ và độ ẩm</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="menu5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Thoát</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tìm kiếm </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchInfo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>49</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableHistory">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>841</width>
      <height>381</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thời gian vào</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thời gian ra</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>110</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
